aslam_offline_calibration/kalibr/python/exporters/kalibr_rovio_config

```diff
@@ -11,7 +11,29 @@ import rospkg
 from matplotlib.colors import LogNorm
 np.set_printoptions(suppress=True)
 
-def printCameraBlock(camConfig, T_SC, q_SC):
+
+def list_fill_with(l, min_size, elem):
+    """Fill up array with `elem` to ensure minimum length of `min_size`."""
+    if len(l) >= min_size:
+        return l
+    else:
+        return l + [elem]*(min_size-len(l))
+
+
+def map_dist_model(model_kalibr, params_kalibr):
+    """Map distortion model names and ensure minimum number of elements in parameter list"
+    # Kalibr might write fewer parameters than ROVIO expects, to fill up with 0.0
+    if model_kalibr == "radtan":
+        return "plumb_bob", list_fill_with(params_kalibr, 5, 0.0)
+    elif model_kalibr == "equidistant":
+        return "equidistant", list_fill_with(params_kalibr, 4, 0.0)
+    elif model_kalibr == "fov":
+        return "fov", list_fill_with(params_kalibr, 1, 0.0)
+    else:
+        return model_kalibr, params_kalibr
+
+
+def printCameraBlock(camConfig, T_SC, q_CS):
```

Comment:
Shouldn't this be `t_SC` as well?

Reply:
Shame on me for last minute renaming without testing. The updated version runs for my example. I can squash before merge if needed.